war/pom.xml

```diff
@@ -288,7 +288,7 @@ THE SOFTWARE.
                 <artifactItem>
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>matrix-project</artifactId>
-                  <version>1.18</version>
+                  <version>1.20</version>
```

Comment:
Upgrading Matrix Project to 1.20 also requires upgrading JUnit to 1.47 or higher, but JUnit 1.29 is bundled a few lines below. I suspect this will cause `LoadDetachedPluginsTest` to fail.

Reply:
Thanks for the insights, if the build failed I will have a look next week.

Reply:
As predicted: https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6180/2/testReport/junit/jenkins.install/LoadDetachedPluginsTest/Linux_jdk11___Linux_Build___Test___upgradeFromJenkins1/